ArgoProject/src/client/Client.h

```diff
@@ -0,0 +1,61 @@
+﻿#ifndef _CLIENT_H
+#define _CLIENT_H
+#include"stdafx.h"
+
+namespace app::net
+{
+
+	enum class Packet {
+		CLIENT_NAME,
+		LOBBY_CREATE
+	};
+	class Client
+	{
+	public: // Constructors/Destructor/Assignments
+		Client() = default;
+		~Client() = default;
+
+		Client(Client const &) = default;
+		Client & operator=(Client const &) = default;
+
+		Client(Client &&) = default;
+		Client & operator=(Client &&) = default;
+
+	public: // Public Static Functions
+	public: // Public Member Functions
+		void CloseSocket();
+		bool InitNetwork(std::string const & pIP, int iPort);
+		bool CheckSocket();
+		bool processPacket(Packet _packetType);
+
+
+		//Part of SendGetMethods cpp
+		bool sendAll(char* data, int totalBytes);
```

Comment:
All cases of `char *` can be converted to using `std::byte *`, this will allow it to be more explicit that you are working on bytes instead of characters of a string.
```suggestion
		bool sendAll(std::byte* data, int totalBytes);
```